edx-olive/edx-platform
# SE-1740 Adds rel="noopener" to target="_blank" links

Backports reverse tab-nabbing exploit, cf https://github.com/edx/edx-platform/pull/21635

**JIRA issue**

[SE-1740](https://tasks.opencraft.com/browse/SE-1740)

**Testing instructions**

This change will be deployed to [stage.campus.gov.il](https://stage.campus.gov.il) on Friday AM for testing.

1. Run through our [manual instance checklist](https://gitlab.com/opencraft/documentation/private/blob/master/checklists/manual_instance_test.md) to ensure that pages display ok, and that links and tabs work as expected.

**Author Notes & Concerns**

1. Let me know if you need me to grant your new account staff privileges so you can create courses.
1. [stage.campus.gov.il](https://stage.campus.gov.il) points to a WordPress marketing frontend, which has some broken links to courses which don't exist on stage, and can exhibit other issues.
   The LMS is hosted at [courses.stage.campus.gov.il](https://courses.stage.campus.gov.il), but there are redirects in place which can take you back to the marketing frontend.
   To complete testing, you may have to manually enter direct links to pages in the LMS, like the student dashboard: [courses.stage.campus.gov.il/dashboard](https://courses.stage.campus.gov.il/dashboard).

**Reviewer**
